ds701_book/24-NN-II-Backprop.qmd

```diff
@@ -1425,7 +1460,7 @@ class MLP(Module):
 ```
 
 ```{python}
-#| tags: []
+#| code-fold: false
 # help(Module)
```

Comment:
Why is all this code commented out?

Reply:
I didn't want to create a bunch of output but encouraging students to uncomment and try. Added

"Feel free to uncomment lines of code in the following cell to see the docstrings for each class."

I might still remove this when I consolidate.

Reply:
No problem. I think this is fine.